woof-code/rootfs-packages/ptheme/usr/sbin/ptheme

```diff
@@ -420,6 +421,13 @@ echo -n > $WORKDIR/ptheme-gtkdialog
 echo -n > $WORKDIR/ptheme-pwidgets_profile
 echo -n > $WORKDIR/ptheme-pwidgets_theme
 
+# link cursor themes in /usr/share/icons to $HOME
+while read -r ITHEME; do
+	ETHEME="${ITHEME/\/cursors/}"
+	XTHEME="${ETHEME##*\/}"
+	[ -e "$HOME/.icons/$XTHEME" ] || ln -sv "$ETHEME" $HOME/.icons
```

Comment:
No need to add to /home/spot/.icons too?

Reply:
I suppose so. Pcur already does it so should do it here too.

Reply:
Actually, `pcur -u -t "$PTHEME_MOUSE"` does it..

https://github.com/puppylinux-woof-CE/woof-CE/blob/c5c0e124e3ce9d4f932f53d04fd02655450d8cea/woof-code/rootfs-packages/ptheme/usr/sbin/ptheme#L210